.gitignore

```diff
@@ -1,3 +1,6 @@
 .dir-locals.el
 # Local Netlify folder
-.netlify
\ No newline at end of file
+.netlify
+
+# editor realted
+.idea/
```

Comment:
I think it's a good idea to add all editor-related files to the top-level `.gitignore`, the reason being that I often use the editors in other parts of the project (e.g., web, example apps).